lib/turbograft.rb

```diff
@@ -36,7 +36,7 @@ def referer
 
       ActiveSupport.on_load(:action_view) do
         (ActionView::RoutingUrlFor rescue ActionView::Helpers::UrlHelper).module_eval do
-          include XHRUrlFor
+          prepend XHRUrlFor
         end
       end unless RUBY_VERSION =~ /^1\.8/
```

Comment:
Given we only test on Ruby 2.2, I was planning on bumping the gemspec to that and release `0.3.0`